server/db/models/Product.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Product = db.define('product', {
+  title: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  author: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  description: {
+    type: Sequelize.TEXT,
```

Comment:
Should this one be a string since it's the short description? should we impose a character limit and/or make the default value = a substring of the summary?

Reply:
When I imagine a short description for a book I think of one a unique one-line attention grabber rather than just a piece from the overall summary. But I agree it should be a string.